### PR TITLE
Quick install guide update

### PR DESCRIPTION
**Pull request by Spaceman1984**
Updated quick install guide to be in line with latest Cloudstack version

### CODE REVIEW

**Comment by andrijapanicsb** [comment]
@blueorangutan docbuild

**Comment by andrijapanicsb** [comment]
@blueorangutan docbuild

**Comment by andrijapanicsb** [comment]
@Spaceman1984 you might want to update the following, since it seems we are now on Advanced Zone and VLAN isolation (L2), but we are indeed still on a "flat" single VLAN (i.e. average joe's home/office network) Centos is also 7.9:

_This guide will focus on building a CloudStack cloud using KVM on CentOS 
7.7 with NFS storage on a flat layer-2 network utilizing layer-3 network 
isolation (aka Security Groups), and doing it all on a single piece of 
hardware._

**Comment by andrijapanicsb** [comment]
also check other occurences of "security groups" like e.g. "Security Groups act as distributed firewalls that control access to a group of virtual machines." which are not used any more in this Advanced Zone guide

**Comment by andrijapanicsb** [comment]
Also.

"A /24 network with the gateway being at xxx.xxx.xxx.1, no DHCP should be on this network and none of the computers running CloudStack will have a dynamic address. Again this is done for the sake of simplicity."

can we changed a bit - DHCP is allowed, but we will be setting things statically (/24 or similar network is OK - i.e. any local network home/office network will do)

**Comment by andrijapanicsb** [comment]
@blueorangutan docbuild

**Comment by andrijapanicsb** [comment]
@blueorangutan docbuild

**Comment by andrijapanicsb** [comment]
@blueorangutan docbuild

**Comment by andrijapanicsb** [comment]
@blueorangutan docbuild

File: source/adminguide/api.rst [hunk 1]
```diff
@@ -43,8 +43,8 @@ possible as well. For example, see Using an LDAP Server for User
 Authentication.
 
 
-User Data and Meta Data
------------------------
+User Data and Meta Data via Virtual Router
+------------------------------------------
```
Comment (DaanHoogland) [inline source/adminguide/api.rst:48]:
```suggestion
User Data and Meta Data via the Virtual Router
----------------------------------------------

```

**Comment by andrijapanicsb** [comment]
@blueorangutan docbuild

**Comment by andrijapanicsb** [comment]
@blueorangutan docbuild

**Comment by andrijapanicsb** [comment]
@wido @GabrielBrascher @weizhouapache @kiwiflyer @rhtyd @davidjumani @Pearl1594 @shwstppr @sureshanaparti @borisstoyanov @DennisKonrad @harikrishna-patnala @mike-tutkowski @nathanejohnson @nvazquez  - feel free to review (high-level) the changes to the Quick Install Guide -
 - it's NO MORE with BASIC zone/security groups 
 - it's now an advanced zone, where one can spin everything on a single server/VM/virtualbox (as before), on their (home) network - and the same network is used for both the Pod (i.e. lower part of CIDR range) and for the Public (upper part of CIDR range) - a bit "crazy" setup, but perfectly valid for test/demo purposes on one's laptop/server - this way they can taste the Advanced Zone - AND can access (TCP access) Public side of VR/CPVM/SSVM to test all features of ACS, etc
 - there are numerous warnings that this network setup is purely for demo/test and NOT for production, etc.
 
 I've been running through this guide 2 times, verified every step (blindly following it) - anyone else is also welcome to quickly test, especially that you can spin i.e. VirtualBox with a VM and use it for ACS (this is what I did during my testing)

**Review by andrijapanicsb — APPROVED** [review]
Changed wording in a few places to reflect Advanced Zone
Tested 2 times on VirtualBox

LGTM to be merged

File: source/quickinstallationguide/qig.rst [hunk 2]
```diff
@@ -611,121 +607,194 @@ to using the CloudStack UI for the actual configuration of our cloud.
...
+
+A new Zone wizard will pop-up. Please chose Advanced (don't tick the "Security Groups") and click on Next.
+
+Zone Details
+~~~~~~~~~~~~
 
-A zone is the largest organization entity in CloudStack - and we'll be 
-creating one, this should be the screen that you see in front of you now. And 
-for us there are 5 pieces of information that we need.
+On this page we enter where our dns servers are located.
```
Comment (sureshanaparti) [inline source/quickinstallationguide/qig.rst:641]:
```suggestion
On this page, we enter where our DNS servers are located.
```

File: source/quickinstallationguide/qig.rst [hunk 3]
```diff
@@ -611,121 +607,194 @@ to using the CloudStack UI for the actual configuration of our cloud.
...
+#. Hypervisor - this will be the primary hypervisor used in this zone. In our
+   case, we will select KVM.
+
+Click "Next" to continue.
+
+Physical Network
+~~~~~~~~~~~~~~~~
+There are various network isolation methods supported by Cloudstack. The
+default VLAN option will be sufficient for our purposes. For improved
+performance and/or security, Cloudstack allows different trafic types to run
```
Comment (sureshanaparti) [inline source/quickinstallationguide/qig.rst:671]:
```suggestion
performance and/or security, Cloudstack allows different traffic types to run
```

**Comment by rhtyd** [comment]
@blueorangutan docbuild

**Comment by weizhouapache** [comment]
@andrijapanicsb 
it seems basic zone is still the first option for new cloudstack users...

**Review by rhtyd — APPROVED** [review]
LGTM, reviewed, tested; committed review/proposal by Suresh

**Pull request merged**